### PR TITLE
[DNM] Cross-Repo CI Test PR

### PR DESCRIPTION
**Pull request by artemcm**

### CODE REVIEW

**Comment by artemcm** [comment]
https://github.com/apple/swift-driver/pull/1154
@swift-ci test

**Comment by artemcm** [comment]
https://github.com/apple/swift-driver/pull/1141
@swift-ci test

**Comment by artemcm** [comment]
https://github.com/apple/swift-driver/pull/1141
@swift-ci test

**Comment by artemcm** [comment]
https://github.com/apple/swift-driver/pull/1141
@swift-ci test

**Comment by artemcm** [comment]
https://github.com/apple/swift-driver/pull/1141
@swift-ci test

**Comment by artemcm** [comment]
https://github.com/apple/swift-driver/pull/1141
@swift-ci test

**Comment by artemcm** [comment]
https://github.com/apple/swift-driver/pull/1227
@swift-ci test

**Comment by artemcm** [comment]
https://github.com/apple/llvm-project/pull/5629
https://github.com/apple/swift-driver/pull/1227
@swift-ci test

**Comment by artemcm** [comment]
https://github.com/apple/swift-driver/pull/1227
@swift-ci test

**Comment by artemcm** [comment]
https://github.com/apple/swift-driver/pull/1227
@swift-ci test macOS platform

**Comment by felipepiovezan** [comment]
https://github.com/apple/llvm-project/pull/5792
@swift-ci test

File: utils/build_swift/build_swift/defaults.py [hunk 1]
```diff
@@ -119,7 +119,7 @@ def llvm_install_components():
     platforms.
     """
     components = ['llvm-cov', 'llvm-profdata', 'IndexStore', 'clang',
-                  'clang-resource-headers', 'compiler-rt', 'clangd']
+                  'clang-resource-headers', 'compiler-rt']
```
Comment (artemcm) [inline utils/build_swift/build_swift/defaults.py:122]:
```suggestion
                  'clang-resource-headers', 'compiler-rt']
```

**Comment by artemcm** [comment]
https://github.com/apple/swift-driver/pull/1250
@swift-ci test

**Comment by artemcm** [comment]
https://github.com/apple/swift-driver/pull/1250
@swift-ci test macOS Platform

**Comment by artemcm** [comment]
https://github.com/apple/swift-driver/pull/1259
@swift-ci test

**Comment by artemcm** [comment]
https://github.com/apple/swift-driver/pull/1259
@swift-ci test

**Comment by artemcm** [comment]
https://github.com/apple/swift-driver/pull/1141
@swift-ci test

**Comment by artemcm** [comment]
https://github.com/apple/swift-driver/pull/1259
@swift-ci test

**Comment by artemcm** [comment]
https://github.com/apple/swift-driver/pull/1274
@swift-ci test

**Comment by artemcm** [comment]
https://github.com/apple/swift-driver/pull/1274
@swift-ci Please Test Source Compatibility

**Comment by artemcm** [comment]
https://github.com/apple/swift-driver/pull/1274
@swift-ci test macOS platform

**Comment by artemcm** [comment]
https://github.com/apple/swift-driver/pull/1274
@swift-ci test

**Comment by artemcm** [comment]
https://github.com/apple/swift-driver/pull/1274
@swift-ci Please Test Source Compatibility

**Comment by artemcm** [comment]
https://github.com/apple/swift-driver/pull/1280
@swift-ci smoke test

**Comment by artemcm** [comment]
https://github.com/apple/swift-driver/pull/1274
@swift-ci test

**Comment by artemcm** [comment]
https://github.com/apple/swift-driver/pull/1274
@swift-ci test

**Comment by artemcm** [comment]
@swift-ci Please Build Toolchain macOS Platform

**Comment by artemcm** [comment]
@swift-ci Please Build Toolchain macOS Platform

**Comment by artemcm** [comment]
https://github.com/apple/swift-driver/pull/1274
@swift-ci test

**Comment by artemcm** [comment]
https://github.com/apple/swift-tools-support-core/pull/407
@swift-ci smoke test

**Comment by artemcm** [comment]
https://github.com/apple/swift-driver/pull/1294
@swift-ci smoke test

**Comment by artemcm** [comment]
@swift-ci please build toolchain Windows platform

**Comment by artemcm** [comment]
https://github.com/apple/swift-driver/pull/1345
@swift-ci test

**Comment by compnerd** [comment]
https://github.com/apple/swift-driver/pull/1345
@swift-ci please build toolchain Windows platform

**Comment by compnerd** [comment]
Please test with following PRs:
https://github.com/apple/swift-driver/pull/1345

@swift-ci please build toolchain Windows platform

**Comment by artemcm** [comment]
https://github.com/apple/swift-driver/pull/1350
@swift-ci smoke test

**Comment by artemcm** [comment]
@swift-ci please build toolchain macOS platform

**Pull request closed without merging**